tools/accuracy_checker/accuracy_checker/annotation_converters/ms_coco.py

```diff
@@ -270,13 +306,31 @@ def _create_representations(
         num_iterations = len(image_info)
         progress_reporter = PrintProgressReporter(print_interval=progress_interval)
         progress_reporter.reset(num_iterations, 'annotations')
+        segmentation_colors = (
```

Comment:
I do not think that you need code classes in color. Just label index will be enough and acceptable (less postprocessing required)

Reply:
Fixed